test/cmdlineargs.jl

```diff
@@ -300,37 +300,43 @@ let exename = `$(Base.julia_cmd()) --startup-file=no --color=no`
     @test errors_not_signals(`$exename -C invalidtarget`)
     @test errors_not_signals(`$exename --cpu-target=invalidtarget`)
 
-    # -t, --threads
-    code = "print(Threads.threadpoolsize())"
-    cpu_threads = ccall(:jl_effective_threads, Int32, ())
-    @test string(cpu_threads) ==
-          read(`$exename --threads auto -e $code`, String) ==
-          read(`$exename --threads=auto -e $code`, String) ==
-          read(`$exename -tauto -e $code`, String) ==
-          read(`$exename -t auto -e $code`, String)
-    for nt in (nothing, "1")
-        withenv("JULIA_NUM_THREADS" => nt) do
-            @test read(`$exename --threads=2 -e $code`, String) ==
-                  read(`$exename -t 2 -e $code`, String) == "2"
+    if Sys.iswindows()
```

Comment:
I'm not entirely sure Windows is immune from issues, maybe it just happens less frequently, I seem to remember @vtjnash saying that these tests are a bit unreliable in general (do I remember wrong?).  But it'd also be sad to completely miss this block of tests.

Reply:
I think Windows does not have this segfault issue, but because of that, it might hang instead.